Concentus.Oggfile/OpusOggReadStream.cs

```diff
@@ -70,11 +71,31 @@ public OpusOggReadStream(OpusDecoder decoder, Stream stream)
         /// </summary>
         public long PageGranulePosition { get; private set; }
 
+        /// <summary>
+        /// Gets the sample position in the stream.
+        /// </summary>
+        public long Position => PageGranulePosition * _decoder.NumChannels * BytesPerSample;
+
+        /// <summary>
+        /// Gets the current play time in the stream.
+        /// </summary>
+        public TimeSpan CurrentTime => TimeSpan.FromSeconds(1.0 * PageGranulePosition / _decoder.SampleRate);
```

Comment:
Granule position is always expressed in units of 48000hz regardless of the decoder sample rate so that shouldn't be a variable in the calculation.